pkg/shell/shell.css

```diff
@@ -627,6 +627,16 @@ ul.available-interfaces-group {
     vertical-align: top;
 }
 
+/* workaround: make text red for select if contained in has-error, not in patternfly */
+.has-error > .bootstrap-select > button > span {
+    color: #A94442;
+}
+
+
+.containers-run-message {
```

Comment:
> I think you should just use the 'hidden' attribute on things you want to hide by default.

'hidden' elements will still take up space, whereas 'display: none' will truly be hidden in the rendered page. I use a class for this because then we can reduce in-place styling.